lib/historical_link_checker/error/recurred_for_more_than_one_week_link_error.rb

```diff
@@ -0,0 +1,12 @@
+module HistoricalLinkChecker::Error
+  class RecurredForMoreThanOneWeekLinkError < ::LinkChecker::UriChecker::Error
```

Comment:
Can we shorten the class names at all? I don't think we have a convention of LinkError suffix on other ones.